### PR TITLE
Removed redundant annotation parameters for the following classes

### PR DESCRIPTION
**Pull request by vishalsingh2972**
Checked all the classes and removed redundant annotation parameters from 14 classes

### CODE REVIEW

**Comment by vishalsingh2972** [comment]
> Checked all the classes and removed redundant annotation parameters from 14 classes

Made the additional changes after thorough revision of all redundant parameters @Tonypsilon

File: src/main/java/de/tonypsilon/bmm/backend/match/data/Match.java [hunk 1]
```diff
@@ -43,17 +43,17 @@ public class Match {
      * Note: To avoid strange round off behaviour for half points, points are stored
      * at the doubled amount to guarantee that they are integers.
      */
-    @Column(name = "overruled_home_board_half_points", unique = false, nullable = true)
```
Comment (Tonypsilon) [inline src/main/java/de/tonypsilon/bmm/backend/match/data/Match.java:46]: nullable = true can be removed here as well.
In IntelliJ, these issues can be found be going right click on the source foulder, -> "analyze" -> "inspect code". In the results tab, the exact locations are shown at Java > Declaration redundancy > Default annotaion parameter value.
image
(Here is how it looks like for me)

File: src/main/java/de/tonypsilon/bmm/backend/organization/data/Organization.java [hunk 2]
```diff
@@ -12,7 +12,7 @@ public class Organization {
     @GeneratedValue(strategy = GenerationType.IDENTITY)
     private Long id;
 
-    @Column(name = "season_id", unique = false, nullable = false)
+    @Column(name = "season_id", nullable = false)
     private Long seasonId;
 
     @Column(unique = false, nullable = false)
```
Comment (Tonypsilon) [inline src/main/java/de/tonypsilon/bmm/backend/organization/data/Organization.java]: unique=false can be removed

File: src/main/java/de/tonypsilon/bmm/backend/match/data/Match.java (hunk 1 above)
Comment (vishalsingh2972) [reply src/main/java/de/tonypsilon/bmm/backend/match/data/Match.java:46]: Thanks for this @Tonypsilon , didn't know about this until now

I am using IntelliJ but in 2 classes having Default annotation parameter value, have removed those and commited
image

Comment (vishalsingh2972) [reply src/main/java/de/tonypsilon/bmm/backend/match/data/Match.java:46]: Thanks for this @Tonypsilon , didn't know about this until now

Yes I am using IntelliJ , have used "analyze" and removed Default annotation parameter values and committed PR#12

Comment (vishalsingh2972) [reply src/main/java/de/tonypsilon/bmm/backend/match/data/Match.java:46]: Thanks for this @Tonypsilon , didn't know about this until now

Yes I am using IntelliJ , have used "analyze" and removed Default annotation parameter values and committed PR #12

Comment (vishalsingh2972) [reply src/main/java/de/tonypsilon/bmm/backend/match/data/Match.java:46]: Thanks for this @Tonypsilon , didn't know about this until now

Yes I am using IntelliJ , have used "analyze" and removed Default annotation parameter values and committed, have a look!

**Comment by Tonypsilon** [comment]
I merged the PR, thank you for your contribution! :-)

**Pull request merged**